File: python/SimpleArithmetics.py
import random
import time

first_number = random.randint(10,25)
second_number = random.randint(17,35)
result = first_number - second_number
number_of_success = 0
number_of_failure = 0

start_time = time.time()
for count in range (1):
    for counter in range (2):
        user_answer = int(input(f"Enter the result of {first_number} - {second_number} ")) 
        if user_answer == result:
            print("Correct Answer")
            number_of_success += 1
         
        else: 
            print("incorrect Answer \n")
            number_of_failure += 1
         

         
    print("Welcome to the second phase");
    for countt in range (2,4):
        first_number = random.randint(10,55)
        second_number = random.randint(20,60)
        result = first_number - second_number
        user_answer = int(input(f"Enter the result of {first_number} - {second_number} ")) 
        if user_answer == result:
            print("Correct Answer")
            number_of_success += 1
         
        else: 
            print("incorrect Answer \n")
            number_of_failure += 1
  
    print("Welcome to the third phase");
    for counti in range (4,6):
        first_number = random.randint(30,90)
        second_number = random.randint(20,24)
        result = first_number - second_number
        user_answer = int(input(f"Enter the result of {first_number} - {second_number} ")) 
        if user_answer == result:
            print("Correct Answer")
            number_of_success += 1
         
        else: 
            print("incorrect Answer \n")
            number_of_failure += 1



print("Welcome to the fourth phase");
for count in range (1):
    for countin in range (2):
        first_number = random.randint(16,56)
        second_number = random.randint(19,80)
        result = first_number - second_number
        user_answer = int(input(f"Enter the result of {first_number} - {second_number} ")) 
        if user_answer == result:
            print("Correct Answer")
            number_of_success += 1
         
        else: 
            print("incorrect Answer \n")
            number_of_failure += 1


         
    print("Welcome to the fifth phase");
    for counting in range (2,4):
        first_number = random.randint(10,85)
        second_number = random.randint(8,38)
        result = first_number - second_number
        user_answer = int(input(f"Enter the result of {first_number} - {second_number} ")) 
        if user_answer == result:
            print("Correct Answer")
            number_of_success += 1
         
        else: 
            print("incorrect Answer \n")
            number_of_failure += 1
  
    print("Welcome to the sixth phase");
    for counts in range (4,6):
        first_number = random.randint(15,26)
        second_number = random.randint(18,88)
        result = first_number - second_number
        user_answer = int(input(f"Enter the result of {first_number} - {second_number} ")) 
        if user_answer == result:
            print("Correct Answer")
            number_of_success += 1
         
        else: 
            print("incorrect Answer \n")
            number_of_failure += 1


print("Welcome to the seventh phase");
for count in range (1):
    for index in range (2):
        first_number = random.randint(12,44)
        second_number = random.randint(11,60)
        result = first_number - second_number
        user_answer = int(input(f"Enter the result of {first_number} - {second_number} ")) 
        if user_answer == result:
            print("Correct Answer")
            number_of_success += 1
         
        else: 
            print("incorrect Answer \n")
            number_of_failure += 1


         
    print("Welcome to the eight phase");
    for indexx in range (2,4):
        first_number = random.randint(40,88)
        second_number = random.randint(20,35)
        result = first_number - second_number
        user_answer = int(input(f"Enter the result of {first_number} - {second_number} ")) 
        if user_answer == result:
            print("Correct Answer")
            number_of_success += 1
         
        else: 
            print("incorrect Answer \n")
            number_of_failure += 1
  
    print("Welcome to the ninth phase");
    for number in range (4,6):
        first_number = random.randint(12,90)
        second_number = random.randint(15,30)
        result = first_number - second_number
        user_answer = int(input(f"Enter the result of {first_number} - {second_number} ")) 
        if user_answer == result:
            print("Correct Answer")
            number_of_success += 1
         
        else: 
            print("incorrect Answer \n")
            number_of_failure += 1


print(f"Your score is: {number_of_success}")
stop_time = time.time

total_time =  start_time - stop_time

print(f"Total time spent is {total_time/-1000} seconds")
  

File: javascript/SimpleArithmetics.js
        



let result = firstNumber - secondNumber;
        let userAnswer = 0;
        let numberOfSucess = 0;
        let numberOfFailure = 0;
        let startdate = Date.now()
     


    

for(let count = 0; count <= 2; count++){
    
    for(let counter = 0; count < 2; count++){
        var firstNumber = Math.floor(Math.random(10)+500);
        var secondNumber = Math.floor(Math.random(10)+30); 
        let result = firstNumber - secondNumber;
        console.log("Enter the result of: ", firstNumber, " - " ,secondNumber, " : "); 
        const prompt = require("prompt-sync")();
        let neww = prompt("result: ") 
        let userAnswer = parseInt(neww) 

             if(userAnswer == result){
                console.log("Correct Answer");
                numberOfSucess += 1;
             }
             else {
                console.log("incorrect Answer \n");
                numberOfFailure += 1;
             }

    }
     




    console.log("Welcome to the second phase");
    for(let counti = 2; counti < 4; counti++){
        var firstNumber = Math.floor(Math.random(10)+50);
        var secondNumber = Math.floor(Math.random(10)+30); 
         let result = firstNumber - secondNumber;
        console.log("Enter the result of: ", firstNumber, " - " ,secondNumber, " : "); 
        const prompt = require("prompt-sync")();
        let neww = prompt("result: ") 
        let userAnswer = parseInt(neww) 
               77
            if(userAnswer == result){
                console.log("Correct Answer");
                numberOfSucess += 1;
             }
             else {
                 console.log("incorrect Answer");
                 numberOfFailure += 1;
                 
             }
                
               
     }





    console.log("Welcome to the third phase");
    for(let countin = 4; countin < 6; countin ++){
        console.log("Enter the result of firstNumber - secondNumber"); 
        var firstNumber = Math.floor(Math.random(10)+800);
        var secondNumber = Math.floor(Math.random(10)+30); 
         let result = firstNumber - secondNumber;
        console.log("Enter the result of: ", firstNumber, " - " ,secondNumber, " : "); 
        const prompt = require("prompt-sync")();
        let neww = prompt("result: ") 
        let userAnswer = parseInt(neww) 
            if(userAnswer == result){
                console.log("Correct Answer");
                numberOfSucess += 1;
                }
            else {
               console.log("incorrect Answer");
               numberOfFailure += 1;
                 
                }
                
               
             }
}





console.log("Welcome to the fourth phase");
for (let count = 0; count <= 2; count++){
    
        var firstNumber = Math.floor(Math.random(10)+90);
        var secondNumber = Math.floor(Math.random(10)+70); 
         let result = firstNumber - secondNumber;
        console.log("Enter the result of: ", firstNumber, " - " ,secondNumber, " : "); 
        const prompt = require("prompt-sync")();
        let neww = prompt("result: ") 
        let userAnswer = parseInt(neww) 

            if(userAnswer == result){
                console.log("Correct Answer");
                numberOfSucess += 1;
            }
            else {
                console.log("incorrect Answer \n");
                numberOfFailure += 1;
            }
}
         




         console.log("Welcome to the fifth phase");
         for(let counti = 2; counti < 4; counti++){
            var firstNumber = Math.floor(Math.random(10)+60);
            var secondNumber = Math.floor(Math.random(10)+20); 
             let result = firstNumber - secondNumber;
            console.log("Enter the result of: ", firstNumber, " - " ,secondNumber, " : "); 
            const prompt = require("prompt-sync")();
            let neww = prompt("result: ") 
            let userAnswer = parseInt(neww) 
               
                if(userAnswer == result){
                    numberOfSucess += 1;
                    console.log("Correct Answer");
                    
                }
                else {
                   console.log("incorrect Answer");
                    numberOfFailure += 1;
                 
                }
                
               
            }



         console.log("Welcome to the sixth phase");
         for(let countin = 4; countin < 6; countin ++){
            var firstNumber = Math.floor(Math.random(10)+40);
            var secondNumber = Math.floor(Math.random(10)+10);
             let result = firstNumber - secondNumber; 
            console.log("Enter the result of: ", firstNumber, " - " ,secondNumber, " : "); 
            const prompt = require("prompt-sync")();
            let neww = prompt("result: ") 
            let userAnswer = parseInt(neww)  
                 
                if(userAnswer == result){
                   console.log("Correct Answer");
                    numberOfSucess += 1;
                }
                else {
                    console.log("incorrect Answer");
                    numberOfFailure += 1;
                 
                }
                
               
            }




console.log("Welcome to the seventh phase");
for  (let count = 0; count <= 2; count++){
    
        var firstNumber = Math.floor(Math.random(10)+70);
        var secondNumber = Math.floor(Math.random(10)+40); 
         let result = firstNumber - secondNumber;
        console.log("Enter the result of: ", firstNumber, " - " ,secondNumber, " : "); 
        const prompt = require("prompt-sync")();
        let neww = prompt("result: ") 
        let userAnswer = parseInt(neww) 

         if(userAnswer == result){
            console.log("Correct Answer");
            numberOfSucess += 1;
         }
         else {
           console.log("incorrect Answer \n");
            numberOfFailure += 1;
         }
}
       
  
         console.log("Welcome to the eight phase");
         for(let counti = 2; counti < 4; counti++){
            var firstNumber = Math.floor(Math.random(10)+20);
            var secondNumber = Math.floor(Math.random(10)+10); 
             let result = firstNumber - secondNumber;
            console.log("Enter the result of: ", firstNumber, " - " ,secondNumber, " : "); 
            const prompt = require("prompt-sync")();
            let neww = prompt("result: ") 
            let userAnswer = parseInt(neww) 
               
                 
                if(userAnswer == result){
                    numberOfSucess += 1;
                   console.log("Correct Answer");
                   
                }
                else {
                    console.log("incorrect Answer");
                    numberOfFailure += 1;
                 
                }
                
               
            }


         console.log("Welcome to the ninth phase");
         for(let countin = 4; countin < 6; countin ++){
            var firstNumber = Math.floor(Math.random(10)+90);
            var secondNumber = Math.floor(Math.random(10)+60); 
             let result = firstNumber - secondNumber;
            console.log("Enter the result of: ", firstNumber, " - " ,secondNumber, " : "); 
            const prompt = require("prompt-sync")();
            let neww = prompt("result: ") 
            let userAnswer = parseInt(neww) 
               
                if(userAnswer == result){
                    console.log("Correct Answer");
                    numberOfSucess += 1;
                }
                else {
                    console.log("incorrect Answer");
                    numberOfFailure += 1;
                 
                }
                
               
            }



let stopdate = Date.now()
let total = (startdate - stopdate)/-1000
console.log(`Your score is: ${numberOfSucess}` );
console.log(`Total Time Spent is: ${total} seconds` );


